sympy/core/tests/test_symbol.py

```diff
@@ -38,7 +38,6 @@ def test_Dummy_force_dummy_index():
     assert Dummy('d1', dummy_index=2) != Dummy('d2', dummy_index=2)
     d1 = Dummy('d', dummy_index=3)
     d2 = Dummy('d')
-    # not only unlikely, in fact impossible (b/c we keep a list of dummy_index)
```

Comment:
yes that is correct.   The other PR guaranteed no collision in this manner no matter what the use passed in (by keeping a list of all previously created dummy_indexes).